code/modules/reagents/Chemistry-Reagents/Blob-Reagents.dm

```diff
@@ -110,56 +95,17 @@
 	reac_volume = ..()
 	M.apply_damage(0.6*reac_volume, BRUTE)
 
-/datum/reagent/blob/b_sorium
+/datum/reagent/blob/sorium //does brute damage and throws or pushes nearby objects away from the target
 	name = "Sorium"
-	id = "b_sorium"
-	description = ""
+	id = "sorium"
```

Comment:
Change this back to "b_sorium". Otherwise default Sorium made in Chemistry is going to clash with this.